### ISSUE TITLE
Comments on adventures

### ISSUE THREAD

**Issue by Daemon53** [AdventureLookup/AdventureLookup#138]
Could we have a comments section on each adventure for users to discuss the adventure

**Comment by MattColville** [comment]
Thinking about how Amazon does things, it would probably be nice and useful for people to be able to ask questions and get responses from folks who've run the adventure. This would be quite apart from users writing reviews which I also encourage.

**Comment by idiot-sandwich** [comment]
Agreed! I just submitted two adventures and would like to add some of my personal experiences running them, perhaps listing some advice. These are my own opinions and wouldn't fit into a neutral description of the adventure. Reading other people's comments would also have been very valuable to me. 
For instance, new DM's would like to read the weaker points of Lost Mines (deadly first session, main npc to be rescued is roleplayed before being kidnapped, weak motivation on the main villain) so they can adapt where necessary.

The only thing I'd worry about is scope creep; I don't feel adventure lookup is intended as a review or discussion site.
